### ISSUE TITLE
Illegal history operator error with static operations

### ISSUE THREAD

**Issue by nickbattle** [nickbattle/vdmj#3]
If you attempt to call the test operation in the following class, you will get an error:
```
class A
operations
    op: () ==> ()
    op() == skip;

    public static test: () ==> ()
    test() == skip;

sync
    per test => #fin(op) > 0

end A

> p A`test()
Error 4065: Error 4011: Illegal history operator: #fin in 'A' (test.vpp) at line 10:17 in 'A' (test.vpp) at line 10:17
Stopped in 'A' (test.vpp) at line 10:17
10:     per test => #fin(op) > 0
[MainThread-9]> q
```
The problem is that the test operation is static and so it is not associated with any object instance. When the history clause #fin(op) is evaluated, there is no object on which to count the #fin, and op is not itself a static method. This ought to be a type checker error.

**Comment by nickbattle** [comment]
Now this produces a type checking error:
```
Error 3349: Cannot see non-static operation from static context in 'A' (test.vpp) at line 10:22
```

**Comment by nickbattle** [comment]
Fixed in 3.0.1-11
